derive_builder/src/options/mod.rs

```diff
@@ -198,6 +206,9 @@ impl<Mode> OptionsBuilder<Mode> where
                 // setter implicitly enabled
                 self.setter_enabled(true)
             },
+            "try_setter" => {
+                self.try_setter_enabled(true)
```

Comment:
I think the `#[builder(setter(try))] is the better API in the long run, as having a `try_{field}` without the regular setter seems bad.